doc/dev.md

```diff
@@ -85,6 +85,8 @@ DBSession().add(obj)  # add a new object into the DB
 DBSession().commit()  # commit modifications to objects
```

Comment:
Is the part about `DBSession` up to date still?

Reply:
I think so. The DBSession itself still does the same thing. I think I did discuss the case of nested sessions. I think we will probably need to augment the docs once we start updating code to the new scheme.